### PR TITLE
V3-image-section

### PR DESCRIPTION
**Pull request by anemne**
This PR simplifies existing image section by removing title and changes the size and width of the image. 

Screenshot 2024-12-11 at 15 35 21
Screenshot 2024-12-11 at 15 35 28
Screenshot 2024-12-11 at 15 35 38
Screenshot 2024-12-11 at 15 35 46

### CODE REVIEW

File: studio/schemas/documents/pageBuilder.ts [hunk 1]
```diff
@@ -49,9 +46,9 @@ const pageBuilder = defineType({
...
         article,
-        callout,
-        callToAction,
-        testimonals,
+        // callout,
+        // callToAction,
+        // testimonals,
         imageSplitSection,
         imageSection,
         grid,
```
Comment (idamand) [inline studio/schemas/documents/pageBuilder.ts]: Do we use grid or can it be commenter out as well?

**Review by idamand — COMMENTED** [review]
Should we remove title from schema too? :)

File: studio/schemas/documents/pageBuilder.ts (hunk 1 above)
Comment (anemne) [reply studio/schemas/documents/pageBuilder.ts]: Im not sure actually! we might just comment it out for now

**Comment by anemne** [comment]
> Should we remove title from schema too? :)

Yesss

**Review by idamand — APPROVED** [review]
Looks good 👌

**Pull request merged**